SyncMyMod/autoinstall.sh

```diff
@@ -143,32 +133,36 @@ function collectHwInfo {
 ###############################################################################
 # Start mod installation                                                      #
 ###############################################################################
-echo "Starting installation of ${FANCYNAME} v${VERSION}..." > $LOG_FILE
 
+echo "Starting installation of ${FANCYNAME} v${VERSION}..." > "$LOG_FILE"
```

Comment:
FANCYNAME was replaced with APP_NAME, isn't?

Reply:
whops, yes, i forgot to update that line